demisto_sdk/__main__.py

```diff
@@ -2306,6 +2313,8 @@ def create_content_graph(use_docker: bool = False, use_existing: bool = False, o
         use_docker=use_docker,
     ) as content_graph_interface:
         create_content_graph_command(content_graph_interface)
+        if dependencies:
```

Comment:
why is it needed?

Reply:
Easier to debug, to able to create a full graph with dependencies in one command to figure out why the dependencies are calculated